### DISCUSSION TITLE
Have Server side logs in production for next.js app.

### DISCUSSION THREAD

**Discussion by natashajaved** [vercel/next.js#23605] [Help]
I'm fairly new to next.js. In development mode we can see the server side logs where our server runs but in production I wanna access those logs as well without actually accessing the server. Is there a way I could the server logs in browser or in some file and look through them as needed

**Comment by OlegPuzankin** [comment]
What kind of server do you use for production deployment? If you use Vercel hosting you can see all your server-side logs directly on the dashboard.
